Configs/.local/lib/hyde/hyprsunset.sh

```diff
@@ -1,42 +1,108 @@
-#! /bin/env bash
```

Comment:
Only use #/bin/env sh if you're sure it's POSIX compliant, so linters won't shout. Otherwise use #/bin/env bash when your script is actually bash.


Also let's use the conventional #/bin/env instead of /usr/bin/env.